components/layout/ContextBar.tsx

```
@@ -0,0 +1,56 @@
+          return (
+            <button
+              key={s.id}
+              onClick={() => { document.getElementById(s.id)?.scrollIntoView({ behavior:'smooth', block:'start' }); track({ name:'navigation_select', section: s.id }) }}
+              className={cn(
+                'relative px-4 h-8 rounded-lg text-[13px] font-semibold tracking-wide transition-all duration-200 focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-[var(--color-accent-ring)]',
+                current 
+                  ? 'bg-[var(--color-accent)]/10 text-[var(--color-accent)] border border-[var(--color-accent)]/20 shadow-sm' 
+                  : 'text-[var(--text-color-muted)] hover:text-white hover:bg-white/8 border border-transparent'
+              )}
+              aria-current={current ? 'true' : undefined}
+            >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)